pilot/pkg/networking/plugin/authz/authorization.go

```diff
@@ -47,7 +47,14 @@ func NewBuilder(actionType ActionType, push *model.PushContext, proxy *model.Pro
 	option := builder.Option{
 		IsCustomBuilder: actionType == Custom,
 	}
-	policies := push.AuthzPolicies.ListAuthorizationPolicies(proxy.ConfigNamespace, proxy.Labels)
+
+	selectionOpts := model.WorkloadSelectionOpts{
+		Namespace:      proxy.ConfigNamespace,
+		WorkloadLabels: proxy.Labels,
+	}
+
+	policies := push.AuthzPolicies.ListAuthorizationPolicies(selectionOpts)
+
```

Comment:
Nit: You added some blank lines within the code, but haven't done it anywhere else within this PR.